source/fuzz/fuzzer_pass_add_bit_instruction_synonyms.cpp

```diff
@@ -64,6 +64,14 @@ void FuzzerPassAddBitInstructionSynonyms::Apply() {
           continue;
         }
 
+        // Make sure fuzzer never applies a transformation to a bitwise
+        // instruction with differently signed operands.
+        if (TransformationAddBitInstructionSynonym::
```

Comment:
I think we can also move the two `if` statements above into this function (they are duplicated in the `IsApplicable` as well).

Reply:
Done.